### DISCUSSION TITLE
How to use a pre-trained model to train on my data or use my data + voxceleb1 data set to train

### DISCUSSION THREAD

**Discussion by kevin00616** [speechbrain/speechbrain#2238] [General]
My training is to train the ECAPA-TDNN model.

I have two questions want to ask.

1.How can I add my own data to the voxceleb1 data set for training?
2.How can I use the pre-trained model of ECAPA-TDNN to train on my own data set? (meaning:Fine Tuning)

**Comment by bowiesimon9** [comment]
1. look up the voxceleb1 data set format, and make your data the same format, then add your data to it.
2. if you can train the model with voxceleb1 data, then replace voxceleb1 with your data, make sure your data format the same as voxceleb1 data.
3. maybe you should describe your problem with more details.

**Comment by kevin00616** [comment]
@bowiesimon9 When I try to add the name and folder location of my audio file in voxceleb1's train.csv, 
I cannot enter it because the number of data rows has reached the maximum, so I cannot add new rows.

Then I attempted to use the filenames from VoxCeleb1 for several files, replacing the content with my audio files. However, I still encountered some issues that prevented proper training.
